pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java

```diff
@@ -1253,6 +1254,25 @@ public static void verifyHybridTableConfigs(String rawTableName, TableConfig off
           "Time column names are different for table: %s! Offline time column name: %s. Realtime time column name: %s",
           rawTableName, offlineTimeColumnName, realtimeTimeColumnName));
     }
+    TenantConfig offlineTenantConfig = offlineTableConfig.getTenantConfig();
+    TenantConfig realtimeTenantConfig = realtimeTableConfig.getTenantConfig();
+    Preconditions.checkNotNull(realtimeTenantConfig,
+        "Found null realtime table tenant config in hybrid table check for table: %s", rawTableName);
+    Preconditions.checkNotNull(offlineTableConfig,
+        "Found null offline table tenant config in hybrid table check for table: %s", rawTableName);
+    String offlineBroker = offlineTenantConfig.getBroker();
+    String realtimeBroker = realtimeTenantConfig.getBroker();
+    if (offlineBroker == null || realtimeBroker == null) {
```

Comment:
This is allowed for single tenant cluster. We should treat `null` as `TagNameUtils.DEFAULT_TENANT_NAME`

Reply:
@Jackie-Jiang i treat null as TagNameUtils.DEFAULT_TENANT_NAME. 
some of units test can fail if i don't explicitly set DEFAULT_TENANT_NAME since default realtime broker and offline brokers might not the same. For now i kept the unit tests (passed extra broker in the tableConfig) and reverted quickstart test changes

Reply:
I don't follow how is that possible. Can you double check? I feel it might be a typo based on the exception

Reply:
@Jackie-Jiang i think i didn't explain it clearly. I reverted the TableConfigsRestletResourceTest and integrationTests. 

I will keep PinotTableRestletResourceTest and TableConfigUtilsTest since they are needed. what do you think?